Makefile

```diff
@@ -50,109 +51,127 @@ export CGO_ENABLED=0
-endif
-ifeq ($(NO_DOCKER),)
-unit: GOTESTFLAGS:=$(GOTESTFLAGS) --tags=example
-endif
+## unit: Run unit tests
 unit: out
-	@echo "> Running unit/integration tests..."
+	@echo "=====> Running unit/integration tests..."
 	$(GOCMD) test $(GOTESTFLAGS) -timeout=$(UNIT_TIMEOUT) ./...
 
+## acceptance: Run acceptance tests
 acceptance: out
-	@echo "> Running acceptance tests..."
+	@echo "=====> Running acceptance tests..."
 	$(GOCMD) test $(GOTESTFLAGS) -timeout=$(ACCEPTANCE_TIMEOUT) -tags=acceptance ./acceptance
 
+## acceptance-all: Run all acceptance tests
 acceptance-all: export ACCEPTANCE_SUITE_CONFIG:=$(shell $(CAT) .$/acceptance$/testconfig$/all.json)
 acceptance-all:
-	@echo "> Running acceptance tests..."
+	@echo "=====> Running acceptance tests..."
 	$(GOCMD) test $(GOTESTFLAGS) -timeout=$(ACCEPTANCE_TIMEOUT) -tags=acceptance ./acceptance
 
-clean:
-	@echo "> Cleaning workspace..."
-	@$(RMRF) .$/out benchmarks.test || (exit 0)
+## prepare-for-pr: Run clean, verify, and test operations and check for uncommitted changes
+prepare-for-pr: tidy verify test
+	@git diff-index --quiet HEAD -- ||\\
+	(echo "-----------------" &&\\
+	echo "NOTICE: There are some files that have not been committed." &&\\
+	echo "-----------------\\n" &&\\
+	git status &&\\
+	echo "\\n-----------------" &&\\
+	echo "NOTICE: There are some files that have not been committed." &&\\
+	echo "-----------------\\n"  &&\\
+	exit 0)
 
+## verify: Run format and lint checks
 verify: verify-format lint
 
-generate: install-mockgen
-	@echo "> Generating mocks..."
-	$(GOCMD) generate ./...
-
+## verify-format: Verify the format
 verify-format: install-goimports
-	@echo "> Verifying format..."
+	@echo "=====> Verifying format..."
 	$(if $(shell goimports -l -local ${PACKAGE_BASE} ${SRC}), @echo ERROR: Format verification failed! && goimports ${GOIMPORTS_DIFF_OPTION} -local ${PACKAGE_BASE} ${SRC} && exit 1)
 
-prepare-for-pr: tidy verify test
-	@git diff-index --quiet HEAD -- ||\
-	(echo "-----------------" &&\
-	echo "NOTICE: There are some files that have not been committed." &&\
-	echo "-----------------\n" &&\
-	git status &&\
-	echo "\n-----------------" &&\
-	echo "NOTICE: There are some files that have not been committed." &&\
-	echo "-----------------\n"  &&\
-	exit 0)
-
+## benchmark: Run benchmark tests
 benchmark: out
-	@echo "> Running Benchmarks"
+	@echo "=====> Running benchmarks"
 	$(GOCMD) test -run=^$  -bench=. -benchtime=1s -benchmem -memprofile=./out/bench_mem.out -cpuprofile=./out/bench_cpu.out -tags=benchmarks ./benchmarks/ -v
-# NOTE: You can analyze the results, using go tool pprof. For instance, you can start a server to see a graph of the cpu usage by running
-# go tool pprof -http=":8082" out/bench_cpu.out. Alternatively, you can run go tool pprof, and in the ensuing cli, run
-# commands like top10 or web to dig down into the cpu and memory usage
-# For more, see https://blog.golang.org/pprof
```

Comment:
As above, I think that this can be a useful note to leave in the file.

Reply:
Yes, I'm sorry I shouldn't have dropped it. Thanks for reminding me